ci-cd/bin/get-hippo-groups-yaml

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
```

Comment:
Could do with some commenting, I think. It may be just because I'm not a bash expert, but I find it hard to follow this.